arch/arm/src/stm32h7/hardware/stm32h7x3xx_pinmap.h

```diff
@@ -737,20 +737,20 @@
 #define GPIO_OTGHS_ID             (GPIO_ALT|GPIO_AF12|GPIO_SPEED_100MHz|GPIO_OPENDRAIN|GPIO_PULLUP|GPIO_PORTB|GPIO_PIN12)
 #define GPIO_OTGHS_SOF            (GPIO_ALT|GPIO_AF12|GPIO_SPEED_100MHz|GPIO_PUSHPULL|GPIO_FLOAT|GPIO_PORTA|GPIO_PIN4)
 
-#define GPIO_OTG_HS_ULPI_CK       (GPIO_ALT|GPIO_AF10|GPIO_PORTA|GPIO_PIN5)
-#define GPIO_OTG_HS_ULPI_D0       (GPIO_ALT|GPIO_AF10|GPIO_PORTA|GPIO_PIN3)
-#define GPIO_OTG_HS_ULPI_D1       (GPIO_ALT|GPIO_AF10|GPIO_PORTB|GPIO_PIN0)
-#define GPIO_OTG_HS_ULPI_D2       (GPIO_ALT|GPIO_AF10|GPIO_PORTB|GPIO_PIN1)
-#define GPIO_OTG_HS_ULPI_D3       (GPIO_ALT|GPIO_AF10|GPIO_PORTB|GPIO_PIN10)
-#define GPIO_OTG_HS_ULPI_D4       (GPIO_ALT|GPIO_AF10|GPIO_PORTB|GPIO_PIN11)
-#define GPIO_OTG_HS_ULPI_D5       (GPIO_ALT|GPIO_AF10|GPIO_PORTB|GPIO_PIN12)
-#define GPIO_OTG_HS_ULPI_D6       (GPIO_ALT|GPIO_AF10|GPIO_PORTB|GPIO_PIN13)
-#define GPIO_OTG_HS_ULPI_D7       (GPIO_ALT|GPIO_AF10|GPIO_PORTB|GPIO_PIN5)
-#define GPIO_OTG_HS_ULPI_DIR_1    (GPIO_ALT|GPIO_AF10|GPIO_PORTC|GPIO_PIN2)
-#define GPIO_OTG_HS_ULPI_DIR_2    (GPIO_ALT|GPIO_AF10|GPIO_PORTI|GPIO_PIN11)
-#define GPIO_OTG_HS_ULPI_NXT_1    (GPIO_ALT|GPIO_AF10|GPIO_PORTC|GPIO_PIN3)
-#define GPIO_OTG_HS_ULPI_NXT_2    (GPIO_ALT|GPIO_AF10|GPIO_PORTH|GPIO_PIN4)
-#define GPIO_OTG_HS_ULPI_STP      (GPIO_ALT|GPIO_AF10|GPIO_PORTC|GPIO_PIN0)
+#define GPIO_OTG_HS_ULPI_CK       (GPIO_ALT|GPIO_AF10|GPIO_SPEED_100MHz|GPIO_PUSHPULL|GPIO_PORTA|GPIO_PIN5)
```

Comment:
I realize theses defines lack a _n. 

But  speed/slew rate is really board layout specific. I fell it would be better to add the _n suffix and then add the speed setting in the board.h.

Reply:
True. ULPI needs 60MHz clock, so I assumed that `GPIO_SPEED_100MHz` is the only working setting.
This `GPIO_SPEED_100MHz` is really misleading as it has nothing to do with the maximum GPIO frequency. We should change it some day to make it compatible with the manual (`Low speed`, `Very high speed`, etc)